scripts/compare_all_artifacts.py

```diff
@@ -13,6 +13,14 @@ def parse_arguments():
         type=str,
         help="Current gcc hash",
     )
+    parser.add_argument(
+        "-project",
+        "--project",
+        required=False,
+        default="gcc",
+        type=str,
+        help="Project to test",
+    )
```

Comment:
Since only valid choices are gcc and binutils, `choices = ["gcc", "binutils"]` might be a good validation method.

Reply:
added choices